constants/version.go

```diff
@@ -0,0 +1,3 @@
+package constants
+
+var Version string = "source"
```

Comment:
Does this get overwritten, or is it source? IMO, if it’s piped in via LDflags, best to have an “empty” var with a comment of “piped in via LDflags on build”